config/routes.rb

```diff
@@ -1,2 +1,6 @@
 Rails.application.routes.draw do
+  get 'pages/about'
+
+  get 'pages/contact'
+
```

Comment:
`Rubocop` is complaining because `Extra empty line detected at block body end.`, so `delete` that line and `rubocop` should be happy!